ndb/src/google/cloud/ndb/model.py

```diff
@@ -3676,6 +3607,32 @@ class a different name when stored in Google Cloud Datastore than the
         """
         return cls.__name__
 
+    @classmethod
+    def _lookup_model(cls, kind, default_model=None):
+        """Get the model class for  the given kind.
+
+        Args:
+            kind (str): The name of the kind to look up.
+            default_model: The model class to return if the kind can't be
```

Comment:
```suggestion
            default_model (Optional[type]): The model class to return if the kind can't be
```

May have 80-char issues (GitHub suggestion doesn't support multi-line yet)

Reply:
7e49c1c